src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -51,10 +50,15 @@ public void testCollect() {
         metrics = collector.collect();
         assertTrue(metrics.isEmpty());
 
-        // Adding a non-numeric metric does nothing
-        collector.addMetric(buildNonNumericMetric("name2", "group"));
+        // Adding a non-numeric metric converted
+        KafkaMetric nonNumericMetric = buildNonNumericMetric("name", "group");
+        collector.addMetric(nonNumericMetric);
         metrics = collector.collect();
-        assertTrue(metrics.isEmpty());
+        assertEquals(1, metrics.size());
+        assertEquals("kafka_server_group_name", metrics.get(0).name);
+        assertEquals(1, metrics.get(0).samples.size());
+        assertEquals(1.0, metrics.get(0).samples.get(0).value, 1.0);
```

Comment:
Use 1.0 for the delta/epsilon looks odd.   That's a floating point tolerance value.  0 should work in this use-case because there is no floating point maths involved.

Reply:
Value set to 0.1 like in other test assertions